### DISCUSSION TITLE
[Bug] CLI Tool throwing an error after finishing cookie audit

### DISCUSSION THREAD

**Discussion by zinkohlaing** [GoogleChromeLabs/ps-analysis-tool#634] [Support Forum] [answered]
Hi,

I am trying to use CLI Tool to audit the URLs contained in the CSV file.

It seems like it has finished the cookie audit but encountered some error after that.

Is there anything that I need to do (e.g. format CSV file or format the URLs) to make sure it goes through?

Thanks!

![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/13776707/da8fba5f-958a-40e9-a7d0-eb1637aa8d6e)

**Comment by milindmore22** [comment]
Hello @zinkohlaing 

Thanks for the bug report. Yes, the cookie analysis has been completed, but the next step is to analyze the tech stack of a website.

Please make sure to add valid URL’s in the CSV file. Furthermore, please share the CSV file you used so that we can test things on our end.

Meanwhile, you can bypass technology analysis by adding the `-nt` flag to your command, please check the [PSAT CLI wiki](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface) for more details.

Here’s an example: `npm run cli -- -c /pathto/input_url.csv -nt`

By using `-nt`, you’re essentially telling the PSAT CLI to skip the technology analysis step.

**Comment by zinkohlaing** [comment]
Hi @milindmore22 I don't want to share the file here, is there an email address that I should send to?

I will try to skip the technology analysis part and see how it goes!

**Comment by milindmore22** [comment]
Hello @zinkohlaing 

Please share your file using the [google form](https://forms.gle/ejQX5dtUu9GPx79HA). When you choose the “PSAT CLI” option in the form, you'll be able to upload your file in the next section.

Please note that you will need to log in using google account to submit the form, as it's a requirement for file upload.

If you encounter any problems while submitting the form, please don't hesitate to let me know.

**Comment by zinkohlaing** [comment]
Thanks, I have uploaded the file via the form

**Comment by milindmore22** [comment, marked as answer]
Hi @zinkohlaing,

Thanks for submitting the form and letting us analyze your CSV file! We identified that Imperva Incapsula or a similar protection service might be affecting the PSAT CLI analysis.

Here are two options to consider:

**Bypass Incapsula Protection (if possible):** You can try contacting Imperva Incapsula support to see if there’s a way to temporarily disable protection for the URLs you need PSAT to analyze.

**Use the PSAT Extension:** You can use the PSAT Chrome extension, as it won’t be affected by protection services.

**Comment by zinkohlaing** [comment]
Thanks for your reply!

Actually if I disable technology audit, my cookie scan goes through though.
So I should be ok now.

**Comment by milindmore22** [comment]
That's a great solution! I'm pleased to hear you found a workaround.  To keep track of this discussion, I'll convert this issue to a GitHub discussion.  If you have any further questions, feel free to create a [new support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum).
